src/components/InstallModalContent.js

```diff
@@ -23,14 +43,55 @@ export class InstallModalContent extends React.Component {
 
         <h3 className="install-title">Install the Chaos Experiments</h3>
 
-        <div className="modal-install-instruction"><span className="modal-install-instruction-number"></span>You can install the Chaos Experiments by following command</div>
-        <div className="modal-code">kubectl create -f  {this.props.expcrdurl}</div>
-        <div className="modal-install-instruction"><span className="modal-install-instruction-number gray-text-color">Notes:</span></div>
+        <div className="modal-install-instruction">
+          <span className="modal-install-instruction-number"></span>You can
+          install the Chaos Experiments by following command
+        </div>
+        <div className="modal-install-code-row">
+          <div className="modal-code">{this.command()}
+          {document.queryCommandSupported("copy") && (
+            <Popover
+              // Allows the popover to show over the modal
+              containerStyle={{ zIndex: 1000 }}
+              isOpen={this.state.copied}
+              position={"bottom"}
+              content={({ position, targetRect, popoverRect }) => (
+                <ArrowContainer
+                  position={position}
+                  targetRect={targetRect}
+                  popoverRect={popoverRect}
+                  arrowColor="#b0b0b0"
+                  arrowSize={3}
+                  arrowStyle={{ opacity: 0.7 }}
+                >
+                  <div className="modal-install-popover-content">Copied</div>
+                </ArrowContainer>
+              )}
+            >
+              <button
+                onClick={this.copyToClipboard}
+                className="modal-install-clipboard"
+              >
+                <FaCopy />
+              </button>
+            </Popover>
+          )}</div>
+        </div>
+        <div className="modal-install-instruction">
+          <span className="modal-install-instruction-number gray-text-color">
+            Notes:
+          </span>
+        </div>
 
-        <div className="modal-install-instruction"><span className="modal-install-instruction-number"></span><a href="https://github.com/litmuschaos/chaos-operator/blob/master/README.md">Install Litmus Operator, </a>a tool for for injecting chaos experiments on Kubernetes</div>
+        <div className="modal-install-instruction">
+          <span className="modal-install-instruction-number"></span>
+          <a href="https://github.com/litmuschaos/chaos-operator/blob/master/README.md">
+            Install Litmus Operator
+          </a>
+          a tool for for injecting chaos experiments on Kubernetes
```

Comment:
give space in between `Install Litmus operator` and  `a tool for for injecting chaos experiments on Kubernetes`
![demo](https://user-images.githubusercontent.com/33670159/73913317-b5e65e80-48dc-11ea-8d52-aeaef73ca580.png)
And remove `for` from line no. 91